src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Extensions/ControlExtensionsSamplePage.xaml

```diff
@@ -104,6 +106,24 @@
 					</extensions:ControlExtensions.Icon>
 				</ComboBox>
 			</smtx:XamlDisplay>
+
+			<TextBlock Text="ControlExtensions.AlternateContent"
+					   Style="{StaticResource MaterialHeadline5}" />
+
+			<TextBlock Text="This property can be used in order to have an alternate content for ToggleButton for example:"
+					   Style="{StaticResource MaterialBody1}" />
+
+			<smtx:XamlDisplay UniqueKey="ControlExtensionsSamplePage_ToggleButton"
```

Comment:
We could have different varieties here, like: (text: "Dark mode" vs "Light mode")

Reply:
good idea, will add an example for it